spec/ama_layout/decorators/navigation_decorator_spec.rb

```diff
@@ -129,4 +138,53 @@
       expect(navigation_presenter.account_toggle).to eq "render"
     end
   end
+
+  context 'notification center' do
+    let(:any?) { true }
+    let(:user) { OpenStruct.new(navigation: 'member', notifications: notifications) }
+    let(:notifications) { OpenStruct.new(active: [notification], all: [notification], any?: any?) }
```

Comment:
Could you also cover some "0" and "2" count cases? If the `active` and `all` arrays are variables, you won't have to override `any?` either.